src/Features/CSharp/Portable/InlineDeclaration/CSharpInlineDeclarationCodeFixProvider.cs

```diff
@@ -160,22 +170,30 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
             var declarationExpression = GetDeclarationExpression(
                 sourceText, identifier, newType, singleDeclarator ? null : declarator);
 
+            return (local, identifier, declarationExpression, invocationOrCreation);
+        }
+
+        private SyntaxNode ReplaceNode(SemanticModel semanticModel, SyntaxNode currentRoot, (ILocalSymbol, IdentifierNameSyntax, DeclarationExpressionSyntax, SyntaxNode) t, SyntaxNode currentNode)
```

Comment:
't' is not a good name here :)

Also, instead of passing the whole tuple *consider* (i.e. up to you) just passing the data you need.